src/pages/Vote/index.tsx

```diff
@@ -229,7 +230,7 @@ export default function Vote() {
           )
         })}
       </TopSection>
-      <TYPE.subHeader color="text3">{t('votePage.minimumThreshold')}</TYPE.subHeader>
+      <TYPE.subHeader color="text3">{t('votePage.pngPerAvax', { pngSymbol: CHAINS[chainId].png_symbol! })}</TYPE.subHeader>
```

Comment:
is there any reason to use `pngPerAvax` which doesn't exist in `votePage` object